Sources/BraintreePayPalNativeCheckout/BTPayPalNativeError.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/**
+ Error returned from the native PayPal flow
+ */
+@objc public enum BTPayPalNativeError: Int, LocalizedError {
```

Comment:
We had taken a slightly different approach to errors previously ([Amex swift errors](https://github.com/braintree/braintree_ios/blob/next-major-version/Sources/BraintreeAmericanExpress/BTAmericanExpressError.swift)). Is there a reason to prefer `Int` + `LocalizedError` or is it just a matter of preference?

Reply:
Went this route based on Sammy's old branch - happy to use whatever you prefer. I'll update to be inline with the Amex errors.

Reply:
I do think that conforming to `Int` gets the `errorCode` for free